.travis.yml

```diff
@@ -17,6 +18,7 @@ matrix:
     fast_finish: true
 
 before_install:
+  - echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```

Comment:
`if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini fi;`

Or maybe just set the memory limit for `"$TRAVIS_PHP_VERSION" == "5.3" `